src/ColorContextMenu-Core.package/RadialMenuItemMorph.class/instance/setBaseColorForLabel..st

```diff
@@ -0,0 +1,16 @@
+accessing
```

Comment:
Tbh this is the kind of behavior I would like not to see in `RadialMenuItemMorph`, which should not be aware of the concrete domain of color choosing. Can we please extract this to `ColorChooserMorph`?